main/src/ca/uwaterloo/flix/api/Bootstrap.scala

```diff
@@ -621,4 +629,68 @@ class Bootstrap(val projectPath: Path, apiKey: Option[String]) {
         }
     }
   }
+
+  /**
+    * Package the current project and release it on GitHub.
+    */
+  def release(flix: Flix): Validation[Unit, BootstrapError] = {
+    val formatter = flix.getFormatter
```

Comment:
I understand you want the formatter for pretty colors, but lets keep focus on drop the colors for now.

Reply:
I have created a separate ticket to deal with colors: https://github.com/flix/flix/issues/6974 where we can apply it everywhere instead of just do this particular feature.

Reply:
But we're already using it in the package manager. Couldn't we just leave it in?

Reply:
No, because I I think we need to do it differently.

Reply:
Also for the errors?

Reply:
But thats a completely different code path and there we use colors consistently everywhere.

I am not arguing against the feature. I am arguing for having PRs that add specific functionality :)